.oscal/ssp.oscal.yaml

```
@@ -0,0 +1,289 @@
+system-security-plan:
+  uuid: 8c1ee2a6-bd14-4f77-b2f7-3d2bc0aaa55e
+  metadata:
+    title: "Oscal Server Software Development System Security Plan"
+    last-modified: "2024-10-24T00:00:00Z"
+    version: "1.0"
+    oscal-version: "1.0.4"
+    published: "2024-10-24T00:00:00Z"
+    roles:
+      - id: authorizing-official
+        title: Authorizing Official
+      - id: system-owner
+        title: System Owner
+      - id: information-security-officer
+        title: Information System Security Officer
+      - id: developer
+        title: Software Developer
+      - id: security-engineer
+        title: Security Engineer
+      - id: prepared-by
+        title: Document Preparer
+      - id: prepared-for
+        title: Document Recipient
+    
+    parties:
+      - uuid: 11111111-0000-4000-9000-100000000001
+        type: organization
+        name: "Metaschema Framework Organization"
+        short-name: "MFO"
+      - uuid: 11111111-0000-4000-9000-100000000002
+        type: person
+        name: "David Waltermire"
+        props:
+          - name: job-title
+            value: Software Architect
+      - uuid: 22222222-0000-4000-9000-100000000002
+        type: person
+        name: "Paul Wand"
+        props:
+          - name: job-title
+            value: Senior Software Architect
+      - uuid: 88888888-0000-4000-9000-100000000008
+        type: person
+        name: "John Doe"
+        props:
+          - name: job-title
+            value: Document Preparer
+      - uuid: 99999999-0000-4000-9000-100000000009
+        type: organization
+        name: "FedRAMP PMO"
+        addresses:
+          - addr-lines:
+              - "1234 FedRAMP Way"
+            city: "Washington"
+            state: "DC"
+            postal-code: "20001"
```

Blocking: perhaps put the website instead of a fake address?  We also do not call it the FedRAMP PMO anymore, just FedRAMP. More importantly, what does FedRAMP as an organization party do in the context of the project's system security plan? If you keep it, I would propose these changes.

```suggestion
      - uuid: 99999999-0000-4000-9000-100000000009
        type: organization
        name:  Federal Risk and Authorization Management Program        
        short-name: FedRAMP
        link:
          href: https://fedramp.gov
          rel: homepage
```

However, I would prefer we define FedRAMP's usage of the SSP in FedRAMP, and not explicitly code in the FedRAMP role here.

So, I would more strongly prefer we delete it.

```suggestion
```